### ISSUE TITLE
Waiting time issue

### ISSUE THREAD

**Issue by jatinthapar1910** [TablePlus/TablePlus#2575]
### Please answer the questions below, it helps us to keep track of the issue. Please do not ignore it or your issue will be closed.

1. Which driver are you using and version of it (Ex: PostgreSQL 10.0): Postgres 12.6

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81): 396

3. The steps to reproduce this issue:

I find a major issue with Table Plus.

I have made a connection with Postgres server and whenever I execute a query it takes like minutes as waiting time but the actual query execution time is always less. Ex. Lets say I execute a query and it takes like 300 seconds to get the data where execution time is just 1 second. 

I tried to use PgAdmin for the same server but I don’t see any such issues. 

Something is definitely wrong with Table Plus.

**Comment by huyphams** [comment]
I don't think that this issue is related to TablePlus, are you using the same user/credentials? The connection is made directly or over SSH?

**Comment by Dmitriy-teamcmp** [comment]
Have the similar problem. Query is not running on the server, but GUI is still waiting for the response. If I click Cancel button no query appears in the popup. Clicking the Refresh button does not help. Same problem had with previous versions.
MariaDB 10.5.12
TablePlus: Version 5.0.2 (458)

**Comment by Dmitriy-teamcmp** [comment]
Have the similar problem. Query is not running on the server, but GUI is still waiting for the response. If I click Cancel button no query appears in the popup. Clicking the Refresh button does not help. Same problem had with previous versions. Similar issue: https://github.com/TablePlus/TablePlus/issues/783
MariaDB 10.5.12
TablePlus: Version 5.0.2 (458)
